src/runner/test-run-controller.ts

```diff
@@ -24,9 +25,13 @@ interface AttemptResult {
 
 class Quarantine {
     public attempts: TestRunErrorFormattableAdapter[][];
+    public TEST_RUN_THRESHOLD: number;
+    public PASSED_QUARANTINE_THRESHOLD: number;
 
     public constructor () {
         this.attempts = [];
+        this.TEST_RUN_THRESHOLD = DEFAULT_TEST_RUN_THRESHOLD;
```

Comment:
Please make not constant members in camelCase.
You can make this members as properies with getter/setter since you already have special set functions for them